src/mcu.cpp

```diff
@@ -1180,9 +1191,25 @@ int main(int argc, char *argv[])
             {
                 port = atoi(argv[i] + 3);
             }
-            else if (!strncmp(argv[i], "-a:", 3))
+            else if (!strncmp(argv[i], "-d:", 3))
```

Comment:
any reason to change this? and what benefit of doing strchr below?

Reply:
I first tried to merge -a with -ab. It didn't go very cleanly, so I separated them back, acvidentally turning -a to -d(from "device"). Oops...